.github/workflows/build.yml

```
@@ -27,49 +27,165 @@ jobs:
+        retention-days: 3
+
+  # ═══════════════════════════════════════════════════════════════════════
+  # Phase 2: macOS builds  (starts after Linux finishes, ~136 min avg)
+  # ═══════════════════════════════════════════════════════════════════════
+  build-macos:
+    needs: build-linux
+    if: (!cancelled())
+    permissions: {}
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJson(needs.generate-matrix.outputs.macos-matrix) }}
```

actionlint (1.7.12)

[error] property "generate-matrix" is not defined in object type {build-linux: {outputs: {}; result: string}}

(expression)